### PR TITLE
Added my profile to Link Free

### PR DESCRIPTION
**Pull request by ibrahimrehman1**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

linkfree-profile-add

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1578"></a>

### CODE REVIEW

File: public/data/ibrahimrehman1.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Remote Work",
+      "date": "April 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started working at ConceptZi in part-time, as a Full Stack Web and Mobile App Developer. Mostly working on React, Vue and Flutter",
+      "url": "https://www.eddiejaoude.io/"
```
Comment (kumarsonsoff3) [inline public/data/ibrahimrehman1.json:30]: Are you sure, you wanted to add Eddie's URL here?
I think you should update it with your URL or you can safely remove line no. `30`, as it's not compulsory to add a URL to milestones! 



__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please revert the changes from the `package-lock.json` file, and make the suggested changes below! Let us know if you need any help with this! 🛠

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

File: public/data/ibrahimrehman1.json (hunk 1 above)
Comment (ibrahimrehman1) [reply public/data/ibrahimrehman1.json]: Can you please check now? I have made the changes.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for contributing! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Pull request merged**